src/core/config.py

```
@@ -10,0 +1,57 @@
+"""
+MVP Configuration for AIQLeads
+"""
+from typing import Dict, Any
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)